cyhy_commander/commander.py

```diff
@@ -595,11 +593,10 @@ def do_work(self):
             config_section, SHUTDOWN_WHEN_IDLE
         )
         self.__logger.info("Idle shutdown: %s", self.__shutdown_when_idle)
-        default_owner = config.get(config_section, DEFAULT_OWNER)
-        self.__logger.info('Default owner: "%s"' % default_owner)
+        self.__logger.info('Default owner: "%s"' % DEFAULT_OWNER)
         default_scheduler = config.get(config_section, DEFAULT_SCHEDULER)
         self.__logger.info('Default scheduler: "%s"' % default_scheduler)
-        self.__setup_default_owner(default_owner, default_scheduler)
+        self.__setup_default_owner(DEFAULT_OWNER, default_scheduler)
```

Comment:
Is `DEFAULT_OWNER` defined here?  I think I'm missing something.

Reply:
See https://github.com/cisagov/cyhy-core/pull/101/commits/ae965a9eac512ec6ac1c379ef4a0e2c41da4777b for the `DEFAULT_OWNER` definition.